test/sol6/kyberDao.js

```diff
@@ -409,6 +410,17 @@ contract('KyberDao', function(accounts) {
         tInPrecision: new BN(tInPrecision),
         link: link
       });
+      let eventLogs;
+      for (let i = 0; i < txResult.logs.length; i++) {
+          if (txResult.logs[i].event == 'NewCampaignCreated') {
+              eventLogs = txResult.logs[i];
+              break;
+          }
+      }
+      Helper.assertEqual(eventLogs.args.options.length, options.length);
```

Comment:
can use maybe the array equal API Quang added?

Reply:
updated